pkg/test/helpers.go

```diff
@@ -260,6 +260,51 @@ func FormatVersionList(versions []*v1.Version) string {
 	}`, len(versions), len(versions), versionJson.String())
 }
 
+func FormatMachineTypeList(mt []*v1.MachineType) string {
+	var mtJson bytes.Buffer
+
+	v1.MarshalMachineTypeList(mt, &mtJson)
+
+	return fmt.Sprintf(`
+	{
+		"kind": "MachineTypeList",
+		"page": 1,
+		"size": %d,
+		"total": %d,
+		"items": %s
```

Comment:
Kind of a nitpick, but we can move this into it's own backtick string and re-use:

```
var listString string = ` // Feel free to rename
{
    "kind": "%s",
    "page": 1,
    "size": %d,
    "total": %d,
    "items": %s
}`
```